### ISSUE TITLE
What's your relationship to the upstream project?

### ISSUE THREAD

**Issue by ams-tschoening** [RealityRipple/squirrelmail#8]
I need to host an arbitrary old instance of SquirrelMail originally not set up by me at all. Though, I needed to update that host from UB 16.04 to 20.04 because I needed a newer PHP and ran into some problems with SquirrelMail, e.g. the INBOX simply doesn't show any mails for anyone anymore. OTOH, sending mails still seems to work. This is most likely because my version of Squirrel doesn't support the newer PHP properly, e.g. I get the following error:

```
Kategorie: | PHP
-- | --
Nachricht: | Illegal string offset 'A001'
FILE: | /[...]/squirrelmail/functions/imap_general.php
LINE: | 456
```

During investigation of how to deal with problems like these I had a look for newer releases, forks etc. of SquirrelMail and came across your repo. While there hasn't been any release for years, it seems their is at least some maintenance UPSTREAM and in your repo of course. What I don't understand yet is how your repo and UPSTREAM are connected, if at all and which code base I'm should be using in future. So would be great if you could explain a bit about your background, motivation and stuff and answer questions like the following:

Am I correct that your repo is NOT an official GitHub-mirror, fork of the project or alike, but you are simply an interested person like myself trying to keep things running? If so, are you providing your fixes relevant to other users UPSTREAM? Are you merging ALL changes made UPSTREAM or only some of interest to you? Are you releasing in some sort or simply providing your code as-is to people who might find it helpful?

Thanks!

**Comment by RealityRipple** [comment]
- The 'trunk' branch contains an exact copy of https://sourceforge.net/p/squirrelmail/code/HEAD/tree/trunk/squirrelmail/. While not an official mirror, it's essentially equal to Upstream (though delayed by up to a day when changes to the official trunk are made).
- The 'master' branch contains the changes listed in the Readme, and receives merges from 'trunk' regularly.
- The third branch, 'rrs-website' should be ignored, it's for my own website.

There are no copies of official releases or other branches from Upstream. Only trunk (v1.5.2).

I have no plans to make any permanent releases, however the 'master' branch should always be at least functionally stable (or as stable as Upstream trunk, anyway). I also have no relation to the official SquirrelMail team, though I've tinkered with and used SquirrelMail for about a decade now. The fixes I've added have not been sent Upstream, though apparently they've noticed a few incompatibilities and have started making similar fixes (see everything added to 'trunk' since February this year).
